helm/cluster-operator/templates/deployment.yaml

```diff
@@ -23,11 +23,9 @@ spec:
           preferredDuringSchedulingIgnoredDuringExecution:
           - podAffinityTerm:
               labelSelector:
-                matchExpressions:
```

Comment:
I got same question around this; https://github.com/giantswarm/cluster-operator/pull/832#discussion_r352582289